docs/elements.md

```diff
@@ -447,9 +447,10 @@ Element to arrange given blocks of code or text that are displayed initially in
+| `feedback`            | "none", "first-wrong", or "first-wrong-verbose"                | "none"                         | The level of feedback the student will recieve upon giving an incorrect answer. Available with the `dag` or `ranking` grading mode. `first-wrong` will tell the student which block in their answer was the first to be incorrect, `none` will give no feedback. If set to "first-wrong-verbose", if the first incorrect block is a distractor any feedback associated with that distractor will be shown as well (see "distractor-feedback" in `<pl-answer>`)                                                                                                                                         |.
 | `format`              | "code" or "default"                    | "default"                      | If this property is set to "code", then the contents of each of the blocks will be wrapped with a `pl-code` element.                                                                                                                                                                                                                                                                                     |
 | `code-language`       | string                                 | -                              | The programming language syntax highlighting to use. Only available when using `format="code"`.                                                                                                                                                                                                                                                                                                          |
+| `distractor-info`       | "show-distractors" or "show-feedback"                                 | -                              | The level of feedback that is to be given to students with respect to the distractors. `show-distractors` will show students which blocks were incorrect after the problem has been solved or all attempts have been sued. `show-feedback` shows the same but includes the feedback strings associated with each distractors as we by the `distractor-feedback` attribute for each `<pl-answer>` block.                                                                                                                                                                                                                                                                                                          |
```

Comment:
To make sure I understand, here we are using `feedback` to control the feedback the student is getting while working on the problem, and then `distractor-info` is used to control feedback they are shown once they have completed the problem or used all attempts?

Reply:
@CoffeePoweredComputers I don't think there is any point in the "show-distractors" option here---if the student has completed the question, they can see the correct solution. So they are already shown (by them being omitted from the correct) which blocks are the distractor blocks, its just more work for no reason to figure it out if we don't directly show them.

With "show-feedback", it seems like if the question writer goes to the trouble of writing the feedback, they are going to want to show it. 

Given all the above, can we just get rid of this `distractor-info` property completely and just show it all by default?

Reply:
Re: the "first-wrong" and "first-wrong-verbose" options above:

There are at least 3 different ways that the line can be wrong:
- the current block group hasn't been finished
- the dependencies of the line haven't been met (based on the depends attribute"
- the line is a distractor

currently, the 'first-wrong' feedback doesn't say anything about why the block is wrong, just that its wrong. If we are going to make 'first-wrong-verbose' be an option, that is going to tell people about if the line is a distractor, should we also have it be telling them about the other 2 possible ways it could be wrong too? or do we want to configure these different things separately? I'm not completely sure myself.

Reply:
I have spend some more time thinking about this and I have decided what I want. 

I think that we should leave `feedback` exactly as @CoffeePoweredComputers implemented it. I think the differentiation between "first-wrong" and "first-wrong-verbose" will be useful for people, but I don't think anyone will care about having super granular control over feedback related to distractor vs. dependencies haven't been meet vs. block group not finished. If we do want to add more detailed feedback for dependencies not met and for block group not finished, we can add that in sometime in the future and just have it under the same umbrella as "first-wrong-verbose". 

I still think we should get rid of the `distractor-info` attribute altogether and just have it on all the time. If people don't want the feedback to be shown, they won't write any.

Reply:
I talked to @CoffeePoweredComputers in person and we are on the same page about these options. I will go ahead and make the changes.